xiblint/rules/unavailable_custom_classes.py

```diff
@@ -0,0 +1,30 @@
+from xiblint.rules import Rule
+
+
+class UnavailableCustomClasses(Rule):
+    """
+    Ensures a given custom class isn't used and provides a replacement suggestion.
+
+    You must specify a module as part of the class name.
+
+    Example configuration:
+    {
+      "custom_classes": {
+          "SomeModule.LegacyButton": "SomeModule.NewButton"
+      }
+    }
+    """
+    def check(self, context):  # type: (Rule, xiblint.xibcontext.XibContext) -> None
+        unavailable_classes = self.config.get('custom_classes', {})
```

Comment:
now that we have these names maybe the settings keys should mirror them? `unavailable_custom_classes`?

Reply:
It looks like this in JSON:

```json
{
  "rules_config": {
    "unavailable_custom_classes": {
      "custom_classes": {}
    }
}
```

Since `unavailable_custom_classes` could have more options down the road.

Reply:
Sg